lib/bugsnag/exception_notification.rb

```diff
@@ -3,22 +3,22 @@
 
 module ExceptionNotifier
   class BugsnagNotifier
-    def initialize(options=nil, &default_block)
-      @default_options = options
-      @default_block = default_block
+    def initialize(options=nil)
+      @default_options = options || {}
     end
 
     def call(exception, options={}, &block)
-      merged_block = proc do |notification|
-        @default_block.call(notification) if @default_block
-        block.call(notification) if block
-      end
+      options = @default_options.merge(options) if options
```

Comment:
I guess it breaks when `options` is `nil`.

Reply:
Surely. I will fix it!